src/rewrite/vct/rewrite/EncodeArrayValues.scala

```diff
@@ -52,14 +52,24 @@ case object EncodeArrayValues extends RewriterBuilder {
       case other => throw Unreachable(s"Invalid invocation failure: $other")
     }
   }
+
+  case class PointerArrayCreationFailed(arr: NewPointerArray[_]) extends Blame[InvocationFailure] {
+    override def blame(error: InvocationFailure): Unit = error match {
+      case PreconditionFailed(_, _, _) => arr.blame.blame(ArraySize(arr))
+      case ContextEverywhereFailedInPre(_, _) => arr.blame.blame(ArraySize(arr)) // Unnecessary?
```

Comment:
Do you want me to change the Unreachable?

I've used Unreachable, since the other pieces of code, on which I based this, were also using Unreachable.

Reply:
I'm redoing blames soon so let's just leave it :)